survey-thank-you.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I'll move it once this survey is done, and it will go in that folder forever after that